CHANGELOG.md

```diff
@@ -289,26 +341,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Async helper `aenter_stack` uses `inspect.isfunction` so it will bind lambdas
 
 ## [0.2.0] - 2019-05-23
+
 ### Added
+
 - Support for zip file source
 - Async helper for running tasks concurrently
 - Gitter badge to README
 - Documentation on the Data Flow Facilitator subsystem
 - codesec plugin containing operations which gather security related metrics on
   code and binaries.
 - auth plugin containing an scrypt operation as an example of thread pool usage.
+
 ### Changed
+
 - Standardized the API for most classes in DFFML via inheritance from dffml.base
 - Configuration of classes is now done via the args() and config() methods
 - Documentation is now generated using Sphinx
+
 ### Fixed
+
 - Corrected maxsplit in util.cli.parser
 - Check that dtype is a class in Tensorlfow DNN
 - CI script no longer always exits 0 for plugin tests
 - Corrected render type in setup.py to markdown
 
 ## [0.1.2] - 2019-03-29
+
 ### Added
+
```

Comment:
I think this happened to someone else recently because of their editors config, but let's remove these newlines.

Reply:
Ughh!! I had auto-format on.

Reply:
:( sorry!